backend/src/main.rs

```diff
@@ -104,7 +105,11 @@ async fn main() -> anyhow::Result<()> {
             Ok(()) as anyhow::Result<()>
         };
 
-        futures::try_join!(shutdown_signal, server_f, workers_f, monitor_f)?;
+        let metrics_f =
```

Comment:
we might want to have a way to disable prometheus metrics with env variable because most people will probably not need it and it seems wasteful. We should actually probably not do it unless whitelisted with a flag "PROMETHEUS=1" or something. I use "=true" in other places but now I realize I should probably use =1 and be consistent.

Reply:
Sure, how about having a METRICS_ADDR or something to specify the listen address (I think there probably should be something similar for the address for `windmill::run_server` but maybe that's a separate issue).

Reply:
Sure that's a bit more generic and I guess with that you can chose your network device but it seems like a bit cumbersome to have ppl have to type 0.0.0.0:8001 just to active prometheus.
What about:  match PROMETHEUS:
== 1=> port 8001
== 0 or not defined => no prometheus,
addr@_ =>  addr

if you wanna change all the other ==true to ==1 of other env variables that's great by me, otherwise I will do it in another PR.

Reply:
I don't know if either `1` or `true` is more consistent than the other. And typing in the address is something I'd do once because shell history, so it's not a big deal. I am certainly onboard with having a reasonable default if users specify a truth value but not an address. But otherwise whatever pattern you have in mind seems fine.

The only thing I feel strongly about, if we're looking at handling environment variables, is switching `DISABLE_*` to `ENABLE_*`. The double negatives for things like `DISABLE_NSJAIL=0` or `(!disable_nsjail)` is a bit much for me personally.

Reply:
:+1:  WDYT about renaming ENABLE_NSAIL to NSJAIL=1 with 1 being the default ?

Reply:
sounds reasonable